src/main/java/org/spongepowered/common/mixin/core/data/types/MixinBlockStoneSlabNewEnumType.java

```diff
@@ -47,4 +53,12 @@
     public String slab$getName() {
         return shadow$getUnlocalizedName();
     }
+
+    public Translation slab$getTranslation() {
```

Comment:
This and all the others really should be done at the end of the constructor and not lazily created like this imo.